### PR TITLE
follow-up #594 (Error Box Styling, remove redudant code in App.tsx) 

### PR DESCRIPTION
**Pull request by T0biii**
follo-wup #594

- Update the Error Box Element to the Styling of Loading 
- Remove Redundant Code in App.tsx

### CODE REVIEW

File: website/src/App.tsx [hunk 1]
```diff
@@ -33,49 +33,24 @@ export const App = observer(class App extends React.Component {
...
+            ) : (
+              !AppState.info ? (
+                <Loading />
+              ) : (
+                <Routes>
+                  <Route path="/" element={<YourDevices />} />
+                  {AppState.info.isAdmin && <Route path="/admin/all-devices" element={<AllDevices />} />}
+                </Routes>
+              )
+            )}
```
Comment (nwaldispuehl) [inline website/src/App.tsx:53]: Cool that we can reduce redundancy; but this nested branching is a bit hard to parse. What if we extract this part into a method like this?
```
...
         <Box component="div" m={2}>
            { this.pageContent() }
          </Box>
...
```
With a function like this:
```
  pageContent() {
    if (AppState.loadingError) {
      return (
          <Error message={AppState.loadingError} />
      );
    }
    else if (AppState.info) {
    ...

  }

```

Comment (T0biii) [reply website/src/App.tsx]: sounds good to me
[`19e2b98` (#605)](https://github.com/freifunkMUC/wg-access-server/pull/605/commits/19e2b98a8dabd1b2a5283cf68670f9f10b3f5cc3)
